tools/validators/instance_validator/tests/fake_instances/GOOD/translation_multi_states.yaml

```diff
@@ -20,21 +20,21 @@ US-SEA-BLDG1-GUID:
 FAN-17-GUID:
   type: HVAC/FAN_SS_VSC
   code: FAN-17
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.